.github/workflows/helm-chart-releaser.yml

```diff
@@ -4,12 +4,12 @@ on:
     branches:
       - 'master'
     paths:
-      - 'install/kubernetes/**'
+      - 'install/helmcharts/**'
```

Comment:
For my understanding, the same level folder name as `deployment_yamls` is the manifest we can deploy in Kubernetes and `helmcharts` is mean to the folder for helm-chart. They both work for deploying to Kubernetes. Is this can make sense?